server/src/domains/orders/orders.service.ts

```diff
@@ -110,10 +113,41 @@ export class OrdersService {
     return this.ordersRepository.save({ ...order, status: cancelStatus });
   }
 
-  async getAll(): Promise<Order[]> {
-    return this.ordersRepository.find({
+  async getAll(
+    query: SearchQueryDto,
+  ): Promise<{ orders: Order[]; totalCount: number }> {
+    const filter: FilterDto = {};
+    const searhcStatus = await this.statusesService.getByValue(query.status);
```

Comment:
Typo in search. Also check out this extension for VScode https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker